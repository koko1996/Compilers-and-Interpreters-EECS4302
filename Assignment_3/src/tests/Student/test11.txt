var p // uninitialized
var q // uninitialized
verify not (p and q)
