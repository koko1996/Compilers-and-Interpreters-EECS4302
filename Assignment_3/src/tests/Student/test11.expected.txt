==========
not ((p) and (q)) is not a tautology.
p=true,q=true
